admin/database/database.go

```diff
@@ -465,9 +479,34 @@ type UpdateServiceOptions struct {
 
 // Usergroup represents a group of org members
 type Usergroup struct {
-	ID    string `db:"id"`
-	OrgID string `db:"org_id"`
-	Name  string `db:"name" validate:"slug"`
+	ID          string         `db:"id"`
+	OrgID       string         `db:"org_id"`
+	Name        string         `db:"name" validate:"slug"`
+	Description sql.NullString `db:"description"`
+	CreatedOn   time.Time      `db:"created_on"`
+	UpdatedOn   time.Time      `db:"updated_on"`
+}
+
+// UsergroupOrgRole represents an organization role of Usergroup
+type UsergroupOrgRole struct {
+	UsergroupID string `db:"usergroup_id"`
+	OrgName     string `db:"org_name"`
+	RoleName    string `db:"role_name"`
+}
+
+// UsergroupProjectRole represents a project role of Usergroup
+type UsergroupProjectRole struct {
+	UsergroupID string `db:"usergroup_id"`
+	ProjectID   string `db:"project_id"`
+	ProjectName string `db:"project_name"`
+	RoleName    string `db:"role_name"`
+}
+
+type UsergroupMember struct {
+	UsergroupID string `db:"usergroup_id"`
+	UserID      string `db:"user_id"`
+	Email       string `db:"email"`
+	DisplayName string `db:"display_name"`
 }
```

Comment:
In line with other comments, I would propose removing this type and using the `Member` type, as is used for org/project member users.